### ISSUE TITLE
Pre-work Checklist: Developer: Samuel Ibarra

### ISSUE THREAD

**Issue by ftblmagician** [hackforla/website#3918]
### Prerequisite
We are looking forward to having you on our team. Please make sure to attend the general Hack for LA onboarding to get the process started https://meetup.com/hackforla/events.

### Overview
As a new developer on the HfLA website team, fill in the following fields as you complete each onboarding item.

### Special Notes
1. It may take you a few weeks to finish this issue, because part of it is learning how to provide updates on issues that take more than a week to complete.    Please keep this issue open until you have been able to do all the steps.  
2. Usually we don't want to you have more than one issue assigned to you at a time, this issue is the exception, because it is instructions on how to work on other issues.
3. The action items listed below should mostly be worked on in a sequential order. However, you don't have to wait on one if you can proceed with the others, For instance, you don't have to wait for attending a weekly meeting before setting up your dev environment.  

### Action Items

- [x] Add yourself to the #hfla-site and #hfla-site-pr Slack channels
- [x] Share your GitHub handle and Gmail address in the hfla-site-onboarding slack channel so you can be added to the Google Drive (alternative to go to meeting and direct message a merge team member at the [meeting](https://github.com/hackforla/website/wiki/Meetings-and-Agendas
))
  - [x] To find contact information for the merge team members and technical leads, please take a look at our [Meet the Team wiki page](https://github.com/hackforla/website/wiki/Meet-the-Team)
- [x] Also, confirm with a merge team member or a technical lead that they have added you to the Google Calendar invites for our Zoom meetings
- [x] (Once added to the Drive) Add yourself to the [team roster](https://docs.google.com/spreadsheets/d/11u71eT-rZTKvVP8Yj_1rKxf2V45GCaFz4AXA7tS_asM/edit#gid=0)
- [x] After you have finished adding yourself to the roster, let a merge team member or a technical lead know you have added yourself to the roster and would like to be added to the `website-write` and `website` teams on GitHub
- [x] Once added to the website-write team:
  - [x] Self Assign this issue (gear in right side panel)
  - [x] Add the "role: front end" or "role: back end" or both label(s) to this issue and remove the "role missing" label (gear in right side panel)
  - [x] Add this issue to the Project Board under the Projects section (gear in right side panel)
- [x] Signup for a [Figma](https://Figma.com) account
- [x] Attend weekly team meetings:
  - [x] Developer (front-end/back-end) weekly team meeting, Tuesdays 7-8pm PST
  - [ ] (Optional) Office Hours, Thursdays 7-8pm PST
  - [ ] All team meeting (UX, Development, Product), Sunday 10am-12pm PST
- [x] Note: There are no meetings on the 1st-7th of every month.
- [x] Note regarding weekly team meeting requirements: All website team members are required to attend at least 1 team meeting in a week (held on Tuesdays, Thursdays and Sundays). In case, you are unable in any given week, you should reach out to the tech leadership team. Exceptions to this requirement may be provided on a case-by-case basis. Also, please let the tech leadership team know (through a slack message in the hfla-site channel as well as an @ mention in a comment of the issue that you would be working on) if you are planning to take a week off or a longer vacation.
- [x] Complete steps 1.1 - 1.6 in [Part 1: Setting up the development environment within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-1-setting-up-the-development-environment)
  - [ ] OPTIONAL: If you run into any issues, use [4.1 How do I ask for help within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#41-what-do-i-do-if-i-need-help) as a visual example to get a hold of our [HfLA website leadership team](https://github.com/hackforla/website/projects/7#card-69730135). That way we can help you resolve any set up issues immediately.
- [x] Read section 2.1 - 2.4 in [Part 2: How the Website team works with GitHub issues within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-2-how-the-website-team-works-with-github-issues)
  - [x] Follow the steps in section [2.3 Where can I find GitHub issues to work on?](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#23-where-can-i-find-github-issues-to-work-on) to assign yourself your first issue under the Project Board's Prioritized Backlog column and use [2.7 Working on a Issue within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#27-working-on-an-issue) to start working on your issue
  - [x] Read [2.6 What to do when you need to stop mid issue](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#26-what-to-do-when-you-need-to-stop-mid-issue)
- [x] Once you take a good first issue, you will provide estimates and progress reports. Also, once you finish providing your "Estimate" action item below, please answer the question in the "Time spent so far" action item (also below).
   - [x] Estimate: Copy the below and put it in the "good first issue" that you picked.
 Check this box when you have completed this task so that we can identify if you understood the instructions and know what to do on all subsequent issues upon assignment. (Note: Please provide estimates on all the issues that you pick up, going forward.)
    ```
   Availability for this week:
   My estimated ETA for completing this issue:
    ```
   - [x] Time spent so far: Copy the question below into a new comment below (in this pre-work issue only) and answer it. This is just to get feedback on how long it took you to get to this point on this pre-work. There is no right or wrong answers. There is no judgement. It is ok if you take a long time or if you do it really fast or at any pace.
    ```
    How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?
    ```
   - [x] Progress Reports: Copy the below and put it in the issue once you have been assigned to the issue at least 5 days (we check weekly on Fridays), or sooner if you have something to report.  If you finish this issue before 5 days are reached, Yeah!!, do it on your next issue. **This update should be done every week for every issue that you are assigned to**. The checkbox here is meant for us to see if you understood the instructions when you end up doing your first weekly progress update.
    ```
    Provide Update
    1. Progress
    2. Blockers
    3. Availability
    4. ETA
    ```
- [x] Read sections 3.1.a - 3.1.c in [3.1 How to make a pull request](https://github.com/hackforla/website/blob/2e3c45fede99e61f479548ccca3f0dda83e351db/CONTRIBUTING.md#31-how-to-make-a-pull-request) to learn more about how to make a pull request (PR) for the issue that you are working on and how to make changes to your PR if changes are requested by the reviewer
- [x] Additional notes:
   - [x] Please don't forget to review pull requests whenever you are able to.  The [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests) guide will teach you how to review pull requests.
   - [x] Please work on only one issue at a time and wait until your pull request is merged before picking up another issue.
- [x] Read and understand how we progress through issues. Then, you can check this off.
Progress through issues with increasing complexity in the following order:
  - Good first issue (one per person)
  - Good second issue (one per person)
  - Small (one per person, with some exceptions, see below)
  - Medium (you can work on more than one medium issue, but only one at a time)
  - Large (you can work on more than one large issue, but only one at a time)
    - The reasons for this progression are:
      - The issues start out as being prescriptive and become less so as you gain more experience by working through increasingly complex issues.
      - We are trying to teach you the team methodology through the issues themselves.
      - It ensures you understand what we expect and the quality of contributions.
    - You can work on back-to-back small issues if it meets the following criteria:
      - You are learning something new and need to work on an issue of a lesser complexity
      - Special request by a lead or pm
- [x] Read the [Start Here - Developers](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=8583%3A0) in Figma
- [x] Go familiarize yourself with the [Hack for LA Design System page in Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3464%3A3)  (where you can see components and their SCSS classes)
- [ ] Once all tasks are completed, close this issue.  This issue will remain open until you get through your first progress report.  Once you have done that, we are confident you know how to keep the momentum going on your issue and keep your team informed.

### What should I do if I have a question about an issue I'm working on, and I haven't gotten a response yet?
- First, you should post the question or blocker as a comment on your assigned issue, so it can be easily referred to in the next bullet points.
- Then, add the label "Status: Help Wanted" so other developers can see it and potentially help answer your question. In addition, you will still need to post a Slack message or bring it up in meeting so we know you need help; see below for how to do that.
- Also, you can post your question on the hfla-site slack channel and link the issue you're working on, so other developers can see and respond.
- Lastly, you can add the issue to the "Development team meeting discussion items" column of the Project Board so that it can be addressed in the next development meeting. Please bring it during the meeting that you need help.

### Resources/Instructions
- [Contributing.md - Hack for LA](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)
- [GitHub Project Board - Hack for LA](https://github.com/hackforla/website/projects/7)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Front End](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Back End/DevOps](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22)
- [Figma - Hack for LA](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website)
- [Google Drive - Hack for LA website team](https://drive.google.com/drive/folders/1p76K0FgfiAWeIIEyoyJ_Iik8FVj8cBjT?usp=sharing)
- [Agenda / Notes - Dev Team Tuesday meeting](https://github.com/hackforla/website/issues/2010)
- [Agenda / Notes - All Team meeting](https://github.com/hackforla/website/issues/2027)
- [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests)

**Comment by ExperimentsInHonesty** [comment]
@ftblmagician How are you doing on setting up your development environment? It looks like you have not yet assigned yourself to a good first issue. If you need help please reach out in the hfla-site slack channel or come to a team meeting.

**Comment by ftblmagician** [comment]
I was able to setup my development environment, but I'm not comfortable with it yet. I also assigned myself a good first issue, and I was able to make the change that was needed. Will there be any meetings held this week? Thank you.

**Comment by ftblmagician** [comment]
How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?

5 hours

**Comment by ExperimentsInHonesty** [comment]
@ftblmagician I noticed that you skipped good first issue and second and went right to small. I unassigned you from #3922. 

Please pick from this category when your development environment has been fully setup, and you are ready to pick up an issue [Good First Issue](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22good+first+issue%22+label%3A%22role%3A+front+end%22+label%3A%22role%3A+back+end%2Fdevops%22#column-7198257)

**Comment by jdingeman** [comment]
Hi @ftblmagician, I left you a message on Slack. You still need to pick up a good first issue and do two of those. I am unassinging you from #2288. 

Please pick from this category
[Good First Issue](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22good+first+issue%22+label%3A%22role%3A+front+end%22+label%3A%22role%3A+back+end%2Fdevops%22#column-7198257)

**Comment by jdingeman** [comment]
Hi @ftblmagician, per our messages on Slack, I can verify that yes you did indeed work on the good first issues. I just want to emphasize that you should avoid unassigning yourself from the issue when it is closed, as we use this for our metrics and activity. 

We apologize for any confusion. 

Please provide us with and update when you can 😄

**Comment by jdingeman** [comment]
Hi @ftblmagician - please provide us with an update

**Comment by roslynwythe** [comment]
Hi @ftblmagician please leave an update (following the instructions in the comment above) so that we know that you would like to continue as part of the HfLA team.  This issue has been flagged for review by the dev lead team because it has been inactive for 2 weeks.  So if you would like to continue, please leave updates at least every two weeks.   

If we do not hear from you by 5/22, we will conclude that you do not want to continue and this issue will be closed.

**Comment by ftblmagician** [comment]
Availability: 2 hours
ETA: 5/28

**Comment by roslynwythe** [comment]
Hi @ftblmagician.  Please create an update as described in the above comment.  It appears that you have not worked on any issues since March 2023.  If you are not currently working on other issues, then we would like to see updates at least every two weeks, to inform us of your progress and any problems you might be experiencing, and to let us know that you are still interested in participating in Hack for LA.  

If we do not hear from you by 7/10/2023, we will assume that you are not interested in participating in Hack for LA at this time, and we will close this issue, but you can reopen it at a later date, leave an update, and move it into the "In Progress" column.  Thank you.

**Comment by ftblmagician** [comment]
Progress: I completed most of what has been asked. I need to go back and review what is left.

Blockers: No blockers

Availability: 2Hr On 7/15

ETA: 7/15

**Comment by t-will-gillis** [comment]
Inactive member; closed due to inactivity.

**Comment by HackforLABot** [comment]
## Activity Log: ftblmagician
### Repo: https://github.com/hackforla/website

#####  ⚠ Important note: The bot updates this comment automatically - do not edit

- ftblmagician opened: https://github.com/hackforla/website/issues/3918 at 01/31/2023, 09:00 PM PST
- ftblmagician assigned: https://github.com/hackforla/website/issues/3918 at 01/31/2023, 09:00 PM PST
- ftblmagician assigned: https://github.com/hackforla/website/issues/3837#event-8453047120 at 02/26/2023, 11:08 AM PST
- ftblmagician commented on: https://github.com/hackforla/website/issues/3918#issuecomment-1445520144 at 02/26/2023, 04:35 PM PST
- ftblmagician commented on: https://github.com/hackforla/website/issues/3837#issuecomment-1448349600 at 02/28/2023, 07:07 AM PST
- ftblmagician unassigned: https://github.com/hackforla/website/issues/3837#issuecomment-1448446187 at 02/28/2023, 08:06 AM PST
- ftblmagician assigned: https://github.com/hackforla/website/issues/4007 at 02/28/2023, 06:46 PM PST
- ftblmagician opened: https://github.com/hackforla/website/pull/4086 at 03/01/2023, 08:06 PM PST
- ftblmagician unassigned: https://github.com/hackforla/website/issues/4007#issuecomment-1449249575 at 03/02/2023, 06:41 AM PST
- ftblmagician commented on: https://github.com/hackforla/website/issues/4007#issuecomment-1454781739 at 03/04/2023, 07:44 AM PST
- ftblmagician commented on: https://github.com/hackforla/website/pull/4086#issuecomment-1454782470 at 03/04/2023, 07:46 AM PST
- ftblmagician commented on: https://github.com/hackforla/website/issues/3918#issuecomment-1459420719 at 03/07/2023, 08:22 PM PST
- ftblmagician PR was merged: https://github.com/hackforla/website/pull/4086#event-8711482107 at 03/09/2023, 10:56 AM PST
- ftblmagician issue closed by PR 4086: https://github.com/hackforla/website/issues/4007#event-8711482335 at 03/09/2023, 10:56 AM PST
- ftblmagician assigned: https://github.com/hackforla/website/issues/4141 at 03/09/2023, 08:04 PM PST
- ftblmagician commented on: https://github.com/hackforla/website/issues/4141#issuecomment-1464926417 at 03/11/2023, 06:44 AM PST
- ftblmagician opened: https://github.com/hackforla/website/pull/4150 at 03/11/2023, 07:13 AM PST
- ftblmagician commented on: https://github.com/hackforla/website/issues/4141#issuecomment-1464932982 at 03/11/2023, 07:17 AM PST
- ftblmagician commented on: https://github.com/hackforla/website/pull/4150#issuecomment-1468258111 at 03/14/2023, 07:57 AM PDT
- ftblmagician unassigned: https://github.com/hackforla/website/issues/4141#issuecomment-1464932982 at 03/14/2023, 07:02 PM PDT
- ftblmagician assigned: https://github.com/hackforla/website/issues/3922#issuecomment-1437720862 at 03/14/2023, 07:15 PM PDT
- ftblmagician PR was merged: https://github.com/hackforla/website/pull/4150#event-8750936531 at 03/14/2023, 09:41 PM PDT
- ftblmagician issue closed by PR 4150: https://github.com/hackforla/website/issues/4141#event-8750936606 at 03/14/2023, 09:41 PM PDT
- ftblmagician commented on: https://github.com/hackforla/website/issues/3922#issuecomment-1477254936 at 03/20/2023, 09:08 PM PDT
- ftblmagician opened: https://github.com/hackforla/website/pull/4278 at 03/24/2023, 11:05 AM PDT
- ftblmagician unassigned: https://github.com/hackforla/website/issues/3922#issuecomment-1484316737 at 03/26/2023, 05:59 PM PDT
- ftblmagician commented on: https://github.com/hackforla/website/pull/4278#issuecomment-1486100887 at 03/27/2023, 07:04 PM PDT
- ftblmagician commented on: https://github.com/hackforla/website/pull/4278#issuecomment-1486104331 at 03/27/2023, 07:10 PM PDT
- ftblmagician assigned: https://github.com/hackforla/website/issues/3922#issuecomment-1484316737 at 03/28/2023, 09:06 PM PDT
- ftblmagician commented on: https://github.com/hackforla/website/pull/4278#issuecomment-1487917125 at 03/28/2023, 09:07 PM PDT
- ftblmagician PR was merged: https://github.com/hackforla/website/pull/4278#event-8881231050 at 03/29/2023, 03:09 PM PDT
- ftblmagician issue closed by PR 4278: https://github.com/hackforla/website/issues/3922#event-8881231199 at 03/29/2023, 03:09 PM PDT
- ftblmagician assigned: https://github.com/hackforla/website/issues/2288#issuecomment-1483559227 at 03/30/2023, 07:31 PM PDT
- ftblmagician assigned: https://github.com/hackforla/website/issues/4007#event-8711482335 at 04/07/2023, 12:25 PM PDT
- ftblmagician assigned: https://github.com/hackforla/website/issues/4141#event-8750936606 at 04/07/2023, 12:27 PM PDT
- ftblmagician commented on: https://github.com/hackforla/website/issues/2288#issuecomment-1501154765 at 04/09/2023, 08:33 AM PDT
- ftblmagician commented on: https://github.com/hackforla/website/issues/2288#issuecomment-1503529561 at 04/11/2023, 07:55 AM PDT
- ftblmagician commented on: https://github.com/hackforla/website/issues/2288#issuecomment-1503533098 at 04/11/2023, 07:56 AM PDT
- ftblmagician opened: https://github.com/hackforla/website/pull/4474 at 04/11/2023, 07:24 PM PDT
- ftblmagician commented on: https://github.com/hackforla/website/pull/4474#issuecomment-1518148386 at 04/21/2023, 10:44 AM PDT
- ftblmagician commented on: https://github.com/hackforla/website/pull/4474#issuecomment-1522673588 at 04/25/2023, 07:29 PM PDT
- ftblmagician commented on: https://github.com/hackforla/website/pull/4474#issuecomment-1530758503 at 05/01/2023, 06:49 PM PDT
- ftblmagician commented on: https://github.com/hackforla/website/pull/4474#issuecomment-1557342442 at 05/22/2023, 07:39 AM PDT
- ftblmagician commented on: https://github.com/hackforla/website/issues/3918#issuecomment-1557345001 at 05/22/2023, 07:41 AM PDT
- ftblmagician commented on: https://github.com/hackforla/website/pull/4474#issuecomment-1567462571 at 05/29/2023, 12:48 PM PDT
- ftblmagician PR was closed: https://github.com/hackforla/website/pull/4474#event-9645440059 at 06/26/2023, 05:13 PM PDT
- ftblmagician unassigned: https://github.com/hackforla/website/issues/2288#issuecomment-1586250052 at 06/26/2023, 05:17 PM PDT
- ftblmagician commented on: https://github.com/hackforla/website/issues/3918#issuecomment-1627764193 at 07/09/2023, 09:24 AM PDT
